app/views/packages/index.html.erb

```diff
@@ -21,4 +38,46 @@
       </div>
   </div>
 
+<script>
+  const unavailableTimes = JSON.parse(document.querySelector('.time-slots').dataset.unavailable)
+  var startTime = ""
+  var endTime = ""
+  const updateBookingTime = (event) => {
+    const slot = event.currentTarget.innerHTML
+    startTime = slot.split("-")[0]
+    endTime = slot.split("-")[1]
+    sessionStorage.setItem("startTime", startTime)
+    sessionStorage.setItem("endTime", endTime)
+
```

Comment:
@yassinmelsir @Matth93 I'm interested here as to how we want to signify that the selection has been made.  The current behavior minimizes the screen after you select the time slot without much notice given to the user that the time slot was successfully chosen.  If you click the package again, you can see that the selection was made but I think we should think about how this is more obvious to the user - lastly, let's keep in mind that the option we land on **does make sense for mobile too :) **

Reply:
Hey I updated the logic, so that the time slot selection menu toggles based on the title and and a minimize button in the selection menu. Also the time slot you have selected is also displayed in the modal header of the photographer booking.